src/assets/css/components/button.css

```diff
@@ -0,0 +1,22 @@
+.btn {
+  color: var(--primary-color);
+  font-family: var(--primary-font);
+  font-size: 16px;
+  font-weight: var(--fw-semibold);
+}
+.btn--secondary {
+  padding: 10px 23px 9px 25px;
+  margin-right: 32px;
+  text-decoration: none;
+  flex-shrink: 0;
+  border-radius: 4px;
+  border: 1px solid #bcd0e5;
+  text-align: center;
+}
+.btn--primary {
+  padding: 10px 23px 9px 25px;
+  text-decoration: none;
+  flex-shrink: 0;
+  border-radius: 4px;
+  background: #02897a;
+}
```

Comment:
I saw the above code from primary and secondary got the duplicate properties like padding, text-decoration, flex-shrink , border-radius. Should we merge them into one ?

Reply:
Yes, We can merge the common properties of the .btn--primary and .btn--secondary classes